ENGAGEHF/Medications/Views/RowContent/DosageGauge/SizePreferences/GaugeSizeKey.swift

```diff
@@ -0,0 +1,19 @@
+//
+// This source file is part of the ENGAGE-HF project based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import Foundation
+import SwiftUI
+
+
+struct GaugeSizeKey: PreferenceKey {
+    static let defaultValue: CGSize = .init(width: 15, height: 15)
```

Comment:
Wondering what the reason for this initial value here is compared to the other ones; does it make sense to either assume realistic values here (e.g. width 350, height 40) or just use `.zero` everywhere?